components/EventDetail.js

```diff
@@ -26,6 +30,16 @@ const StyledDescription = styled.p``;
 
 export default function EventDetail({ event }) {
   const { day, month, year, time } = getDate(event.startDateTime);
+  const router = useRouter();
+
+  function handleDelete() {
+    const ConfirmDeleteMessage = "Are you sure you want to delete the event?";
+    if (confirm(ConfirmDeleteMessage) == true) {
+      deleteEvent(event._id);
+      router.push("/");
+    } else {
+    }
+  }
```

Comment:
- You can put the message outside the component, since it is fixed
- Name the variable in camelCase better
- Even even better, if you plan to have multiple messages in your app, consider storing them in a separate `messages.json` file ^^

Don't forget to mark your function as `async` then!

```js
async function handleDelete() {
    // always prefer early returns
    if (!confirm(confirmDeleteMessage)) {
      return;
    } 
    await deleteEvent(event._id);
    router.push("/");
}
```